### PR TITLE
Adding coredns mixin

### PR DESCRIPTION
**Pull request by paulfantom**

### CODE REVIEW

File: site/content/coredns/_index.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
+---
+title: coredns
+---
+
+## Overview
+
+
```
Comment (povilasv) [inline site/content/coredns/_index.md]: Let's add more context:

CoreDNS mixin provides Grafana dashboard and Prometheus Alerts to monitor CoreDNS. The mixin was introduced in (Kubernetes Node Local DNS Cache blogpost)[https://povilasv.me/kubernetes-node-local-dns-cache/] to better help users monitor CoreDNS in Kubernetes. Mixin can also be used to monitor standalone CoreDNS instance without any orchestrators.

Comment (paulfantom) [reply site/content/coredns/_index.md]: Added :)
Sorry for delay

Comment (paulfantom) [reply site/content/coredns/_index.md]: > Mixin can also be used to monitor standalone CoreDNS instance without any orchestrators.

Yes, but by default it references k8s. More in https://github.com/povilasv/coredns-mixin/pull/5

**Comment by paulfantom** [comment]
I need to figure out how to properly parse URL in mixin description before merging this PR.

**Review by povilasv — APPROVED** [review]

File: site/content/coredns/_index.md (hunk 1 above)
Comment (povilasv) [reply site/content/coredns/_index.md]: That PR got merged ;)

**Comment by povilasv** [comment]
> I need to figure out how to properly parse URL in mixin description before merging this PR.

Awesome, thanks for doing this ;)

**Comment by paulfantom** [comment]
Sorry for delay. Links are working now so I am merging this :)

**Comment by povilasv** [comment]
:tada:

**Pull request merged**